webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/CampaignApiTest.scala

```diff
@@ -135,27 +167,83 @@ class CampaignApiTest extends Specification with AfterAll with Loggable {
       }
     }
 
-    "save one campaign" in {
+    val c1json = {
+      s"""{"info":{
+         |"id":"c1",
+         |"name":"second campaign",
+         |"description":"a test campaign present when rudder boot",
+         |"status":{"value":"enabled"},
+         |"schedule":{"start":{"day":1,"hour":3,"minute":42},"end":{"day":1,"hour":4,"minute":42},"tz":"${tz}","type":"weekly"}
```

Comment:
I'm not sure it's enough. We want: 
- the tz iso offset (which is the only relevant thing for computing times from our pov)
- the ts human name (to display the correct one to the human who selected it)

Reply:
okay so we really want to accept a known timezone ID, and nothing else (not just an offset) : an offset only could be wrong since there are rules for timezone change in some countries ([example in France](https://timezonedb.com/time-zones/Europe/Paris), there is DST and sometimes other future changes in offset)
So I guess we need to refuse everything else in the API, and parse the `ScheduleTimeZone` as a known ID...